joplin/css/utilities/border-box-reset.scss

```diff
@@ -1,8 +1,8 @@
 // https://www.paulirish.com/2012/box-sizing-border-box-ftw/
 
-html {
-  box-sizing: border-box;
-}
-*, *:before, *:after {
-  box-sizing: inherit;
-}
+// html {
```

Comment:
Mateo had this in here...I kinda get what its purpose is, but I don't know that we need it? I'm not a big fan of having a 'utility' that is essentially a global change, bound to have affects we won't think about as typically caused by this thing.